👋 This fixes the broken test for me, but it seems another test in the same class is breaking (may have been breaking all along but wasn't running because of the earlier failing test this PR fixes):

`ReleaseStack_PluginTestJetpack#testPluginActionNotAvailable`

(From a quick glance I think the test might be supplying a dummy site without many fields filled in, and we're now validating more than we were when the test was written.)

```
2021-07-15 17:16:14.206 11883-11921/org.wordpress.android.fluxc.example I/TestRunner: started: testPluginActionNotAvailable(org.wordpress.android.fluxc.release.ReleaseStack_PluginTestJetpack)
....
2021-07-15 17:16:14.595 11883-12449/org.wordpress.android.fluxc.example E/Volley: [95187] NetworkDispatcher.processRequest: Unhandled exception java.lang.IllegalArgumentException: Expected URL scheme 'http' or 'https' but no colon was found
    java.lang.IllegalArgumentException: Expected URL scheme 'http' or 'https' but no colon was found
        at okhttp3.HttpUrl$Builder.parse$okhttp(HttpUrl.kt:1260)
        at okhttp3.HttpUrl$Companion.get(HttpUrl.kt:1633)
        at okhttp3.Request$Builder.url(Request.kt:184)
        at org.wordpress.android.fluxc.network.OkHttpStack.executeRequest(OkHttpStack.java:98)
        at com.android.volley.toolbox.BasicNetwork.performRequest(BasicNetwork.java:123)
        at com.android.volley.NetworkDispatcher.processRequest(NetworkDispatcher.java:131)
        at com.android.volley.NetworkDispatcher.processRequest(NetworkDispatcher.java:111)
        at com.android.volley.NetworkDispatcher.run(NetworkDispatcher.java:90)
2021-07-15 17:16:14.596 11883-11883/org.wordpress.android.fluxc.example E/WordPress-API: Volley error on 
    com.android.volley.VolleyError: java.lang.IllegalArgumentException: Expected URL scheme 'http' or 'https' but no colon was found
        at com.android.volley.NetworkDispatcher.processRequest(NetworkDispatcher.java:163)
        at com.android.volley.NetworkDispatcher.processRequest(NetworkDispatcher.java:111)
        at com.android.volley.NetworkDispatcher.run(NetworkDispatcher.java:90)
     Caused by: java.lang.IllegalArgumentException: Expected URL scheme 'http' or 'https' but no colon was found
        at okhttp3.HttpUrl$Builder.parse$okhttp(HttpUrl.kt:1260)
        at okhttp3.HttpUrl$Companion.get(HttpUrl.kt:1633)
        at okhttp3.Request$Builder.url(Request.kt:184)
        at org.wordpress.android.fluxc.network.OkHttpStack.executeRequest(OkHttpStack.java:98)
        at com.android.volley.toolbox.BasicNetwork.performRequest(BasicNetwork.java:123)
        at com.android.volley.NetworkDispatcher.processRequest(NetworkDispatcher.java:131)
        at com.android.volley.NetworkDispatcher.processRequest(NetworkDispatcher.java:111) 
        at com.android.volley.NetworkDispatcher.run(NetworkDispatcher.java:90) 
2021-07-15 17:16:14.632 11883-11978/org.wordpress.android.fluxc.example E/AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-1
    Process: org.wordpress.android.fluxc.example, PID: 11883
    java.lang.NullPointerException: site.url must not be null
        at org.wordpress.android.fluxc.store.PluginCoroutineStore.executeWPAPIRequest(PluginCoroutineStore.kt:156)
        at org.wordpress.android.fluxc.store.PluginCoroutineStore.syncConfigureSitePlugin(PluginCoroutineStore.kt:103)
        at org.wordpress.android.fluxc.store.PluginCoroutineStore$configureSitePlugin$1.invokeSuspend(PluginCoroutineStore.kt:92)
        at org.wordpress.android.fluxc.store.PluginCoroutineStore$configureSitePlugin$1.invoke(Unknown Source:10)
        at org.wordpress.android.fluxc.tools.CoroutineEngine$launch$1.invokeSuspend(CoroutineEngine.kt:43)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
        at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:738)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
```

--- a/example/src/androidTest/java/org/wordpress/android/fluxc/release/ReleaseStack_PluginTestJetpack.java
+++ b/example/src/androidTest/java/org/wordpress/android/fluxc/release/ReleaseStack_PluginTestJetpack.java
@@ -57,7 +57,6 @@ import static org.junit.Assert.assertNotEquals;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
-import static org.wordpress.android.fluxc.model.SiteModel.ORIGIN_XMLRPC;
 
 public class ReleaseStack_PluginTestJetpack extends ReleaseStack_Base {
     @Inject SiteStore mSiteStore;
@@ -274,50 +273,6 @@ public class ReleaseStack_PluginTestJetpack extends ReleaseStack_Base {
         assertEquals(0, mPluginStore.getPluginDirectory(site, PluginDirectoryType.SITE).size());
 
         signOutWPCom();
-    }
-
-    @Test
-    public void testPluginActionNotAvailable() throws InterruptedException {
-        // Setup some test objects to use in tests
-        String pluginSlug = "doesn't matter";
-        String pluginName = "doesn't matter";
-        SitePluginModel sitePluginModel = new SitePluginModel();
-        sitePluginModel.setName(pluginName);
-        sitePluginModel.setSlug(pluginSlug);
-        ImmutablePluginModel pluginModel = ImmutablePluginModel.newInstance(sitePluginModel, null);
-        assertNotNull(pluginModel);
-
-        SiteModel selfHostedSite = new SiteModel();
-        selfHostedSite.setOrigin(ORIGIN_XMLRPC);
-
-        SiteModel wpComSite = new SiteModel();
-        wpComSite.setIsWPCom(true);
-
-        TestEvents expectedEvent = TestEvents.PLUGIN_ACTION_NOT_AVAILABLE;
-
-        // Test configure plugin
-        ConfigureSitePluginPayload configureSelfHostedSitePlugin =
-                new ConfigureSitePluginPayload(selfHostedSite, pluginName, pluginSlug, true, true);
-        configureSitePlugin(configureSelfHostedSitePlugin, expectedEvent);
-        ConfigureSitePluginPayload configureWPComSitePlugin =
-                new ConfigureSitePluginPayload(wpComSite, pluginName, pluginSlug, true, true);
-        configureSitePlugin(configureWPComSitePlugin, expectedEvent);
-
-        // Test delete plugin
-        deleteSitePlugin(selfHostedSite, pluginModel, expectedEvent);
-        deleteSitePlugin(wpComSite, pluginModel, expectedEvent);
-
-        // Test fetch site plugins
-        fetchSitePlugins(selfHostedSite, expectedEvent);
-        fetchSitePlugins(wpComSite, expectedEvent);
-
-        // Test install plugins
-        installSitePlugin(selfHostedSite, pluginSlug, expectedEvent);
-        installSitePlugin(wpComSite, pluginSlug, expectedEvent);
-
-        // Test update plugins
-        updateSitePlugin(selfHostedSite, pluginName, pluginSlug, expectedEvent);
-        updateSitePlugin(wpComSite, pluginName, pluginSlug, expectedEvent);
     }
 
     @SuppressWarnings("unused")

